ci_framework/roles/cifmw_ceph_spec/vars/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
`vars` directory does not contain any vars currently. We can drop it.

Reply:
done